_posts/2021-08-31-release-1.14.0-rc0.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Help us harden Apache Flink 1.14.0 RC0"
+date:   2021-08-31 00:00:00
+categories: news
+authors:
+- joemoe:
+  name: "Johannes Moser"
+
+---
+
+Dear Flink Community,
+
+We are happy to announce the first release candidate of Apache Flink 1.14. 🎉 
+
+A lot of work from the community went into this release. We worked on improving
+the unified batch and streaming experience, increased the fault tolerance by 
+reducing the in-flight data, worked on connectors,.... as well as tidied some 
+components.
+
+The development cycle is now entering the phase of stabilising the release
+and making it a good, coherent experience for Apache Flink users.
+
+That’s when we want to ask you for help. As you can see on the 
+[1.14. release page](https://cwiki.apache.org/confluence/display/FLINK/1.14+Release)
+a lot of focus went in documentation and testing. We want to ask our users now to 
+get the release candidate and test it, by e.g. running some existing Flink jobs 
+on the release candidate or picking one of the testing issues.
+
+How can you help:
+
+* Get Apache Flink 1.14. release candidate here
+  * The [preview source release and binary convenience releases](https://dist.apache.org/repos/dist/dev/flink/flink-1.14.0-rc0/), 
+  * which are signed with the key with fingerprint [F8E419AA0B60C28879E876859DFF40967ABFC5A4](https://dist.apache.org/repos/dist/release/flink/KEYS).
+  * all artifacts that would normally be deployed to the [Maven Central Repository](https://repository.apache.org/content/repositories/orgapacheflink-1448)
+  * Source code tag ["release-1.14.0-rc0" on Github](https://github.com/apache/flink/releases/tag/release-1.14.0-rc0)
+* Have a look at the list of [testing issues](https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=468&quickFilter=2115)
+* Run some of your existing jobs
+* Provide feedback on [Jira](https://issues.apache.org/jira/projects/FLINK/summary) or on the [user mailing list](https://flink.apache.org/gettinghelp.html#user-mailing-list)
+
+[~50 people](https://github.com/apache/flink/graphs/contributors?from=2021-03-31&to=2021-08-31&type=c) 
```

Comment:
The number of contributors is far less than previous releases, which does not seem right.
I found the following command from the [release process](https://cwiki.apache.org/confluence/display/FLINK/Creating+a+Flink+Release), which gives us 211 contributors from the first 1.14-SNAPSHOT commit to the 1.14.0-rc0.
```
git log --pretty="%an%n%cn" <fromCommit>..<toCommit> | sort | uniq | tr "\n" "," | sed 's/,/, /g'
```